.github/workflows/e2e-suite.yml

```diff
@@ -43,7 +49,42 @@ jobs:
         run: ssh-keygen -t rsa -b 4096 -f ~/.ssh/id_rsa -N ""
 
       - name: run e2e test
-        run: make e2e
+        run: |
+          status=0
+          if ! make e2e > test_output.txt; then
+            echo "EXIT_STATUS=1" >> $GITHUB_ENV
+          fi
+            cat test_output.txt
         env:
           CLEANUP_TEST_LINODE_INSTANCE: true
           LINODE_TOKEN: ${{ secrets.DX_LINODE_TOKEN }}
+          
+      - name: Copy report.xml from hack directory to root directory
```

Comment:
Can we skip this step?

Reply:
i tried with ur suggestion to have hack/filename but it's getting failed